packages/contracts-bedrock/contracts/universal/StandardBridge.sol

```diff
@@ -313,10 +286,13 @@ abstract contract StandardBridge {
         uint256 _amount,
         bytes calldata _extraData
     ) public payable onlyOtherBridge {
-        require(msg.value == _amount, "StandardBridge: amount sent does not match amount required");
-        require(_to != address(this), "StandardBridge: cannot send to self");
```

Comment:
Did you intend to remove `cannot send to self` check?

Reply:
Reverting this change and adding tests for the behavior

Reply:
> I guess we could keep the check here to prevent people from accidentally bricking funds.

Two reactions: 
1. If so, we should do the same in `finalizeBridgeERC20` below.
2. I don't think this check prevents people from losing funds, if they're calling `finalizeBridgeETH` they already lost funds when the initiate bridging on the other side.

Reply:
I guess suggested path forward is that we should either:
1. Not do this check, or
2. Do it on the initiating side.